webviz_subsurface/containers/_summary_stats.py

```diff
@@ -52,69 +80,199 @@ def layout(self):
                                     for i in ['Realizations', 'Statistics']],
                            value='Realizations'),
-            dcc.Graph(id=self.chart_id,
-                      config={
-                          'displaylogo': False,
-                          'modeBarButtonsToRemove': ['sendDataToCloud']
-                      }
-                      )
+            dcc.Checklist(
+                id=self.show_history_uncertainty_id,
+                options=[{'label': 'Show history', 'value': 'SHOW_H'}],
+                values=[],
+            ),
+            html.Div(id=self.chart_id)
         ])
 
     def set_callbacks(self, app):
-        @app.callback(Output(self.chart_id, 'figure'),
+        @app.callback(Output(self.chart_id, 'children'),
                       [Input(self.dropwdown_vector_id, 'value'),
-                       Input(self.radio_plot_type_id, 'value')])
-        def update_plot(vector, summary_plot_type):
+                       Input(self.radio_plot_type_id, 'value'),
+                       Input(self.show_history_uncertainty_id, 'values')])
+        def update_plot(
+                vector,
+                summary_plot_type,
+                show_history_uncertainty_id):
             if summary_plot_type == 'Realizations':
                 return render_realization_plot(
-                    self.ensemble_path,
-                    self.sampling, vector)
+                    ensemble_paths=self.ensemble_paths,
+                    column_keys=self.column_keys,
+                    sampling=self.sampling,
+                    smry_history_columns=self.smry_history_columns,
+                    history_uncertainty=self.history_uncertainty,
+                    vector=vector,
+                    show_history_uncertainty=show_history_uncertainty_id)
             if summary_plot_type == 'Statistics':
                 return render_stat_plot(
-                    self.ensemble_path,
-                    self.sampling, vector)
+                    ensemble_paths=self.ensemble_paths,
+                    column_keys=self.column_keys,
+                    sampling=self.sampling,
+                    vector=vector,
+                    show_history_uncertainty=show_history_uncertainty_id)
 
     def add_webvizstore(self):
-        return [(get_summary_data, [{'ensemble_path': self.ensemble_path,
-                                     'sampling': self.sampling,
-                                     'statistics': False}]),
-                (get_summary_data, [{'ensemble_path': self.ensemble_path,
-                                     'sampling': self.sampling,
-                                     'statistics': True}])]
+        return [(get_summary_data,
+                 [{'ensemble_paths': self.ensemble_paths,
+                   'column_keys': self.column_keys,
+                   'sampling': self.sampling,
+                   'smry_history_columns': self.smry_history_columns,
```

Comment:
This function does not run currently as `get_summary_data` does not take all these arguments